package.json

```diff
@@ -54,7 +54,7 @@
         "build:prod:npm": "BABEL_ENV=production OUTPUT=dist LANGUAGE=en-US REACT=true yarn build:prod:dist",
         "build:prod:storybook": "LANGUAGE=en-US REACT=true BROWSERSLIST_ENV=production BABEL_ENV=development NODE_ENV=development build-storybook -c .storybook -o styleguide/storybook",
         "build:sync": "LANGUAGE=en-US BABEL_ENV=development NODE_ENV=development RSYNC=true webpack --config scripts/webpack.config.js --mode development",
-        "build:variables": "cat `find src/styles/constants -name '*.scss'` > src/styles/variables.scss | node ./scripts/build-style-vars.js src/styles/variables.scss && rm src/styles/variables.scss",
+        "build:variables": "cat `find -s src/styles/constants -name '*.scss'` > src/styles/variables.scss | node ./scripts/build-style-vars.js src/styles/variables.scss && rm src/styles/variables.scss",
```

Comment:
forces sort order to be lexographic, this resolves inconsistency in sort between mac and linux

Reply:
The BSD version of [`find`](https://www.freebsd.org/cgi/man.cgi?find(1)) is **much** more sophisticated than gnu/linux [`find`](http://man7.org/linux/man-pages/man1/find.1.html)...

So `-s` isn't gonna work if circleCI uses linux unfortunately. 

An in-place solution would be to pipe into [`sort`](http://man7.org/linux/man-pages/man1/sort.1.html). Both macOS's BSD version and linux's version support `-n` for numeric sort.

`find src/styles/constants -name '*.scss' | sort -n` or similar should do the trick.

cc @j3tan

Reply:
good eye @booc0mtaco !